RFC-by-stage/1-approved/rename-form-group.md

```diff
@@ -0,0 +1,21 @@
+---
+Status: `Proposal` # Please do not change this.
+Implementer: # It will be changed upon merging and as it moves through the RFC stages
+---
+
+# Rename form group
+
+## The issue to be solved
+
+Currenty GOLD uses the name `form group` to refer to the elements which make up a single form field (label, hint, error message etc). I believe the naming could be improved as it can be easily mistaken for a collection of form fields and not a single form field.
```

Comment:
```suggestion
Currently GOLD uses the name `form group` to refer to the elements which make up a single form field (label, hint, error message etc). I believe the naming could be improved as it can be easily mistaken for a collection of form fields and not a single form field.
```